modules/azure/src/main/java/org/testcontainers/containers/CosmosDBEmulatorContainer.java

```diff
@@ -0,0 +1,57 @@
+package org.testcontainers.containers;
+
+import java.security.KeyStore;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.utility.DockerImageName.parse;
+
+/**
+ * An Azure CosmosDB container
+ */
+public class CosmosDBEmulatorContainer extends GenericContainer<CosmosDBEmulatorContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = parse("mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator");
+
+    private static final int PORT = 8081;
+
+    /**
+     * @param dockerImageName specified docker image name to run
+     */
+    public CosmosDBEmulatorContainer(final DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+        withExposedPorts(PORT);
+        waitingFor(Wait.forLogMessage("(?s).*Started\\r\\n$", 1));
+    }
+
+    /**
+     * @param certificateFilePath path of certificate file in container
+     * @return new KeyStore built with PKCS12
+     */
+    public KeyStore buildNewKeyStore(String certificateFilePath) {
+        return copyFileFromContainer(certificateFilePath,
+                                     inputStream -> {
```

Comment:
nit (codestyle): could you please either add a new line after `copyFileFromContainer(` (so that the args are aligned) or put `inputStream -> {` next to `certificateFilePath` to dramatically reduce the indentation of the block?

Reply:
Done 👍